web/src/components/Accordion.tsx

```diff
@@ -35,7 +35,7 @@ export default function Accordion({
   const Icon: LucideIcon = isTopExpanding === isCollapsed ? ChevronUp : ChevronDown;
 
   return (
-    <div className="flex flex-col gap-1.5 py-1">
+    <section className="flex flex-col gap-1.5 py-1">
```

Comment:
Is section the right one to use here? As far as I can tell `<details>` would be more appropriate to use

Reply:
Details seem to have it's own accordion behaviour that we can't control in a easy way: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details

Section works the same as a div but has more semantic meaning and should always include a heading or distinct set of elements (like buttons with names) which fits our needs here.